public/video-ui/src/slices/checkedFormFields.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice, Action, PayloadAction } from '@reduxjs/toolkit';
+
+export type CheckedFormFieldsState = {
+  [key: string]: Record<string, string | null | undefined>;
```

Comment:
it would be nice to define this once rather than here as `[key: string]: Record<string, string | null | undefined>` and then on line 7 as `Record<string, Record<string, string | null | undefined>>`. 

Furthermore, I think we can be more specific about the type of the state. It appears to be:
`Record<string, Record<string, FieldNotification>>`

Taken from https://github.com/guardian/media-atom-maker/blob/main/public/video-ui/src/constants/FieldNotification.ts